test/150_connect_test.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+. ./config.sh
+
+C1=10.2.1.4
+C2=10.2.1.7
+
+start_suite "Connecting routers after launch"
+
+weave_on $HOST1 launch
+weave_on $HOST2 launch
+
+start_container $HOST1 $C1/24 --name=c1
+start_container $HOST2 $C2/24 --name=c2
+
+assert_raises "exec_on $HOST1 c1 $PING $C2" "1"
+assert_raises "weave_on $HOST2 connect $HOST1"
+assert_raises "exec_on $HOST1 c1 $PING $C2"
+
+end_suite
```

Comment:
Well, you could still test that `weave connect/forget` succeed for multiple args.

Also should test failure cases.

Also just noticed that there is no `forget` test at all. That could be be tested by invoking `forget` followed by `weave stop` and `weave launch` of the other weave, and then checking that there is no connectivity.